Apps/WebClient/src/NewClientApp/src/models/timeline/BcCancerScreeningResultTimelineEntry.ts

```diff
@@ -0,0 +1,44 @@
+// create public class DiagnosticImagingTimelineEntry which extends TimelineEntry
+import { EntryType } from "@/constants/entryType";
+import { DateWrapper, StringISODate } from "@/models/dateWrapper";
+import { CancerScreeningExam } from "@/models/patientDataResponse";
+import TimelineEntry from "@/models/timeline/timelineEntry";
+import { UserComment } from "@/models/userComment";
+
+export default class BcCancerScreeningResultTimelineEntry extends TimelineEntry {
+    public title: string;
+    public documentType: string;
+    public fileId: string;
+    public resultDate: StringISODate;
+    public programName: string;
+
+    private getComments: (entryId: string) => UserComment[] | null;
+
+    public constructor(
+        model: CancerScreeningExam,
+        getComments: (entryId: string) => UserComment[] | null
+    ) {
+        super(
+            model.id ?? `cancerScreening-${model.resultTimestamp}`,
+            EntryType.CancerScreening,
+            new DateWrapper(model.resultTimestamp)
+        );
+
+        this.title = "BC Cancer Result";
```

Comment:
As per previous comment in BcCancerScreeningResultTimelineComponent, could BcCancerScreeningResultTimelineEntry be just CancerScreeningResultTimelineEntry and have "BC Cancer Result" be passed via constructor?